### DISCUSSION TITLE
Uninstall addons from CLI

### DISCUSSION THREAD

**Discussion by sr258** [WeblateOrg/weblate#7308] [Q&A]
I know that you can use the Weblate CLI with the `install_addon` command. I haven't found a corresponding `uninstall_addon` command. Is there are way to uninstall plugins from the CLI to remove them from all components in a projects? I have hundreds of components in the project and I don't want to remove the addons one-by-one.

**Comment by nijel** [comment]
The command line interface is not going to be extended, use API instead: https://docs.weblate.org/en/latest/api.html#add-ons
